protos/NUgus.proto

```diff
@@ -1332,6 +1381,313 @@ PROTO NUgus [
           }
         }
       }
+      Transform {
+        translation 0.000000 0.000000 0.2
+        children [
+          Accelerometer {
+            xAxis TRUE
+            yAxis TRUE
+            zAxis TRUE
+            resolution 0.002
+            lookupTable [ 
+              # MODDELING LUT OF ISS331dlh
+              # Accel (m/s/s) Return Value (RV: -32768 to 32767 (16 bit 2s-compliment))
+              # Noise(% of RV -> ~218ug/root(400(Hz))) 
+              # Noise values calculated such that RV value deviates 35.717
+              # CALCULATION:
+              # 218e-6 * root(400) = 4.36e-3 g of noise
+              # For g = 4, 4.36e-3 is approx 0.109%
+              # Assuming RV worst case Deviation (35.171 or 0.109% of 32768) is equilivant throughout:
+              # Noise in percent = 35.171/RV (% of RV)
+              # NOTE: 
+              # This model assumes noise is equilivant for all values not in saturation
+              # This model assumes that the accelerometer is sampled at 400Hz
+              -4.0625	0	0
```

Comment:
I'm wondering if it might be better to create a docs folder in this repo and put the section of the report we submitted with the explanation of the lookup table creation in there? Or does this comment block explain everything well enough?

Reply:
Yeah that'd be good